jsonnet/telemeter/server/kubernetes/kubernetes.libsonnet

```diff
@@ -99,7 +99,12 @@ local clusterPort = 8081;
       statefulSet.mixin.spec.withPodManagementPolicy('Parallel') +
       statefulSet.mixin.spec.withServiceName('telemeter-server') +
       statefulSet.mixin.spec.template.spec.withServiceAccountName('telemeter-server') +
-      statefulSet.mixin.spec.template.spec.withVolumes([localVolume, tlsVolume]),
+      statefulSet.mixin.spec.template.spec.withVolumes([localVolume, tlsVolume]) +
+      {
+        spec +: {
```

Comment:
could you run `jsonnet fmt` on this file? there should be no space here. I'll make sure that our CI catches jsonnet format error

Reply:
@squat good idea! :+1: PTAL